ccd-definition/AuthorisationCaseField/CareSupervision/eps-managing.json

```diff
@@ -1339,7 +1346,7 @@
   {
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "CARE_SUPERVISION_EPO",
-    "CaseFieldID": "manageDocumentsCourtBundleNoHearingsLabel",
```

Comment:
added the original field to AuthorisationCaseEvent/CareSupervision/DEPRECATED.json